var/spack/repos/builtin/packages/helics/package.py

```diff
@@ -164,6 +165,11 @@ def cmake_args(self):
 
         return args
 
+        # GCC >=13
+        if spec.satisfies("%gcc@13:"):
+            # C++20 required when building with GCC>=13
+            args.append("-DCMAKE_CXX_STANDARD=20")
```

Comment:
This code is unreachable. Checkout the return statement above.

Reply:
Thanks for catching that!